kong/api/routes/kong.lua

```diff
@@ -15,7 +16,7 @@ local lua_version = jit and jit.version or _VERSION
 return {
   ["/"] = {
     GET = function(self, dao, helpers)
-      local distinct_plugins = {}
+      local distinct_plugins = cjson.empty_array
```

Comment:
Did you run a test for this (that is, on a non-empty database)? `cjson.empty_array` is a lightuserdata, which should throw an error when we execute `distinct_plugins[#distinct_plugins+1] = row.name` down below. We should probably use `cjson.empty_array_mt` here instead.

We should also introduce a regression test for this.

Reply:
Good catch.

`cjson.empty_array_mt` fixes it.

I'll try to add tests as well.

But, I've another problem now:

What is `enabled_in_cluster` suppose to mean?
I expected that it represents plugins which are enabled on some api/consumer currently.
But, a plugin shows up in this list even after I delete it.

- GET / -> `enabled_in_cluster : []`
- Create rate-limiting plugin
- GET / -> `enabled_in_cluster : ["rate-limiting"]`
- Delete the plugin
- GET / -> `enabled_in_cluster : ["rate-limiting"]`

Reply:
From https://getkong.org/docs/0.11.x/admin-api/#retrieve-node-information
enabled_in_cluster: Names of plugins that are enabled/configured. That is, the plugins configurations currently in the datastore shared by all Kong nodes

Reply:
Never mind, fixed it.